di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/domain/AuthRequest.java

```diff
@@ -0,0 +1,142 @@
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.CIMIT_STUB_API_KEY;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.CIMIT_STUB_URL;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.CLIENT_ID;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.EVIDENCE_JSON_PAYLOAD;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.F2F_SEND_ERROR_QUEUE;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.F2F_SEND_VC_QUEUE;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.F2F_STUB_QUEUE_NAME;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.FRAUD;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.JSON_PAYLOAD;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.MITIGATED_CIS;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.REQUEST;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.REQUESTED_OAUTH_ERROR;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.REQUESTED_OAUTH_ERROR_DESCRIPTION;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.REQUESTED_OAUTH_ERROR_ENDPOINT;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.REQUESTED_USERINFO_ERROR;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.RESOURCE_ID;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.STRENGTH;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VALIDITY;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VC_NOT_BEFORE_DAY;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VC_NOT_BEFORE_FLAG;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VC_NOT_BEFORE_HOURS;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VC_NOT_BEFORE_MINUTES;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VC_NOT_BEFORE_MONTH;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VC_NOT_BEFORE_SECONDS;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VC_NOT_BEFORE_YEAR;
+import static uk.gov.di.ipv.stub.cred.handlers.RequestParamConstants.VERIFICATION;
+
+@Builder
+@AllArgsConstructor
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AuthRequest {
+    private static final String CHECKED = "checked";
+    private static final String ON = "on";
+    private static final String UTC = "UTC";
+
+    private String clientId;
+    private String request;
+    private String resourceId;
+
+    private String credentialSubjectJson;
+
+    private String evidenceScore;
+    private String validityScore;
+    private String activityScore;
+    private String fraudScore;
+    private String verificationScore;
+    private String biometricVerificationScore;
+    private String evidenceJson;
```

Comment:
We can probably streamline this a bit for the API version - e.g. we don't really need all these individual scores, CIs, etc., we just need the overall evidence JSON block.

It might also be nice to structure some of the other properties (e.g. all the F2F stuff, the error stuff) to make it easier to work with?

No need to overthink it, but it'll be much easier to change now than later!